developers/weaviate/api/graphql/additional-operators.md

```diff
@@ -32,9 +34,11 @@ import GraphQLFiltersExample from '/_includes/code/graphql.filters.example.mdx';
 
 ## Limit argument
 
-Supported by the `Get{}`, `Explore{}` and `Aggregate{}` function.
+The `limit` argument restricts the number of results. These operators support `limit`:
 
-A `limit` argument limits the number of results to a specified positive integer:
+- `Get{}`
```

Comment:
Can we remove the braces here please, since we are moving away from GraphQL-centric syntax? I've also done so on the other pages.